Lib/defconQt/featureTextEditor.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, font=None, parent=None):
         self.editor.modificationChanged.connect(self.setWindowModified)
 
     def setWindowTitle(self, title="Font Features", font=None):
+        title = self.tr("Font Features")
```

Comment:
Alright then remove the intelligence in setWindowTitle, make it

```
self.setWindowTitle(self.tr("Font Features") + " – %s %s" % (
              self.font.info.familyName, self.font.info.styleName))
```

in the caller and just prepend `[*]` in setWindowTitle before calling superclass.